europe/README.EUROPE

```diff
@@ -1,245 +1,273 @@
+
+*Source*: [Lantmäteriet](https://www.lantmateriet.se/en/maps-and-geographic-information/GPS-och-geodetisk-matning/Om-geodesi/Transformationer/Geoidmodeller/)  
+*Format*: GTX  
+*License*: [Creative commons, CC0](https://creativecommons.org/publicdomain/zero/1.0/)
+
+Vertical transformation for Geoid model SWEN17_RH2000. Used to make the
+transitions from heights above sea level in RH 2000 (EPSG:5613) to heights above the
+ellipsoid in SWEREF 99 (EPSG:4377). The RH 2000 height system is used on Swedish mainland, Öland
+and Gotland islands.
+
+* SWEN17_RH2000.gtx
+
+## Included init-files
+
+### Denmark: System definitions related to ETRS89
+
+*Source*: [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Init files with various transformations and conversions of relevance in
+Denmark. All definitions in the init file uses the local ETRS89 reference
+frame as a pivot datum. More information can be found in the
+[NordicTransformations](https://github.com/NordicGeodesy/NordicTransformations)
+repository. While the `.pol` files are init-files they are not supposed to be
+by themselves. The exist purely as helper files for the `DK` init-file.
+The `.pol` files holds polynomial constants used for transformations between
+Danish legacy coordinate reference systems and ETRS89.
+
+* DK
+* DK_bornholm.pol
+* DK_bridges.pol
+* DK_general.pol
+* DK_jutland.pol
+* DK_zealand.pol
+
+### The Faroe Islands: System definitions related to ETRS89
+
+*Source*: [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Init file with various transformations and conversions of relevance in
+The Faroe Islands. All definitions in the init file uses the local ETRS89
+reference frame as a pivot datum. More information can be found in the
+[NordicTransformations](https://github.com/NordicGeodesy/NordicTransformations)
+repository.
+
+* FO
+
+### Nordic+Baltic countries: ITRFxx -> Local ETRS89 realizations
+
+*Source*: [The Nordic Geodetic Commission](https://github.com/NordicGeodesy/NordicTransformations)  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Transformations to and from the common Nordic referenc frame NKG_ETRF00.
+This init-file describes transformations between global reference frames
+and NKG_ETRF00 as well as transformations from NKG_ERTF00 to the local
+realisations of ETRS89 in each of the countries involved with NKG.
+
+All transformations in this init-file uses the common Nordic frame as a
+pivot datum. Exempt from this dogma are transformations with labels
+starting with ``_``. Those transformations are "private" to this file and are
+only used as steps in more complicated transformations.
+
+* NKG
+
+* CHENyx06a.gsb

```

Comment:
This and the following line should be moved under the `### Switzerland: ...` heading - otherwise it looks like they are part of the section for the Nordic+Baltic countries.

Reply:
done.